soc/wch/ch32v/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Michael Hope
```

Comment:
you should probably use/append your name here

Reply:
Since I just split the file into two, I appended my name.